content/en/synthetics/guide/reusing-browser-test-journeys.md

```diff
@@ -0,0 +1,73 @@
+
+The Browser test subtest feature allows you to reuse your journeys among your test suite and consequently:
+* **Saves you time at test creation** (eg. if you have a login test, you just need to call it as a subtest at the beginning of all your test suite, instead of recording the same login steps for each of your tests);
+* **Makes your tests easier to understand**, as you make blocks that are meaningful to the person reading after you;
+* **Allows for better maintenance**, because if your flows change, you only need to update it once, and not once per test.
+
+In the below example, we look into creating and re-using a login subtest.
+
+## Create and re-use a login subtest
+
+If monitoring your application involves first logging into it, a best practice is to create a single test containing all your login steps, and to then have all your other Browser tests leverage that login test as a subtest. 
+
+To create a login test and use it as a subtest in the rest of your test suite:
+
+1. Create a first test A that does nothing but logging into your application. You can set the **start URL** of this test A to your pre login URL.
+
+{{< img src="synthetics/guide/reusing-browser-test-journeys/login_subtest_recording.mp4" alt="Recording the Login subtest" video="true"  width="100%">}}
+
+2. Create a second test B that monitors any post login functionalities of your application. You can set the **start URL** of this test B to your pre login URL as well.
+
+{{< img src="synthetics/guide/reusing-browser-test-journeys/dashboard_test_configuration.png" alt="Configuring the parent test" >}}
+
+In the above example, you can see that this second test focuses on dashboard creation monitoring.
+
+3. Once on the recorder of your test B, click the Subtest button and select the login test A you just created. 
+
+{{< img src="synthetics/guide/reusing-browser-test-journeys/dashboard_test_subtest.mp4" alt="Including subtest in parent test" video="true"  width="100%">}}
+
+You just created a **Subtest** step: all the steps of the subtest (ie. test A) are consequently played at the beginning of your parent test (ie. test B).
+By default, the subtest is played in the main tab. This means your subtest steps are played in the same tab as the previous and following steps. The subtest starts running using the URL that was set in the parent test (ie. the pre login URL), and once all the subtest steps have been executed, the Browser test executes the parent’s first non subtest step from the page the subtest was last on. We did not set any parent step for now.
+
+You can only see that after importing your login subtest (ie. test A) in your parent test (ie. test B), the two variables from the login subtest are imported into your parent test as well.
+
+**Note:** You can choose the tab in which the subtest should execute using [**Subtest Advanced Options**][1].
```

Comment:
```suggestion
    **Note:** You can choose the tab in which the subtest should execute using [**Subtest Advanced Options**][1].
```
Indenting these three paragraphs so that they're part of step 3, not interrupting your ordered list of steps.